hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -257,7 +256,31 @@ protected void clearMapStore() {
         mapDataStore.reset();
     }
 
-    protected void clearIndexes() {
+    public void clearIndexes() {
```

Comment:
Seems like the method may be private.

Reply:
... or at least protected to match the style of other cleanup methods.

Reply:
done